content/copilot/github-copilot-enterprise/copilot-chat-in-github/using-github-copilot-chat-in-githubcom.md

```diff
@@ -146,7 +146,7 @@ You can chat with {% data variables.product.prodname_copilot_short %} about a fi
    - How could I improve this code?
    - How can I test this script?
 
-   If you are aksing about specific lines, you could enter:
+   If you are asking about specific lines, you could enter:
```

Comment:
#### Before
https://docs.github.com/en/enterprise-cloud@latest/copilot/github-copilot-enterprise/copilot-chat-in-github/using-github-copilot-chat-in-githubcom#asking-questions-about-specific-pieces-of-code
<img width="417" alt="image" src="https://github.com/github/docs/assets/2119212/466adfbd-e526-44e4-8a67-8d66caeedff7">

#### After
https://docs-30890-94eb78.preview.ghdocs.com/en/enterprise-cloud@latest/copilot/github-copilot-enterprise/copilot-chat-in-github/using-github-copilot-chat-in-githubcom#asking-questions-about-specific-pieces-of-code
<img width="407" alt="image" src="https://github.com/github/docs/assets/2119212/b31aa992-e6f4-4a0d-9500-7878527a6df2">